### ISSUE TITLE
sysctl in Harvester config does not work

### ISSUE THREAD

**Issue by gitlawr** [harvester/harvester#1405]
**Describe the bug**
1. `os.sysctl` configuration in [docs](https://docs.harvesterhci.io/v0.3-dev/install/harvester-configuration/#ossysctls) and code(which uses sysctls) is inconsistent
2. The value does not apply to OS post-installation.

**To Reproduce**
1. pxe boot with the following values in the config file:
```
os:
  sysctls:
    kernel.domainname: example.com
```
2. ssh to the Harvester node and check:
```
$ sudo sysctl kernel.domainname
kernel.domainname =
```

**Expected behavior**
Configured kernel parameters are applied.

**Environment:**
 - Harvester ISO version: master-9f917060-head

**Comment by bk201** [comment]
I did some tests and found it seems to work but is overridden by netconfig.

Create a test OEM config:
```
# cat /oem/test.yaml
name: Test
stages:
  initramfs:
  - sysctl:
      kernel.domainname: example.com
      net.ipv4.ip_forward: 1
```

Reboot and found:
```
# cat /proc/sys/kernel/domainname

# cat /proc/sys/net/ipv4/ip_forward
1
```

In the network config file `/etc/sysconfig/network/config`:
```
## Type:        string(yes,no,)
## Default:     "yes"
#
# Defines whether to set the default NIS domain. When enabled and no domain
# is provided dynamically or in static settings, /etc/defaultdomain is used.
# Valid values are:
#  - "no" or ""         netconfig does not set the domainname
#  - "yes"              netconfig sets the domainname according to the
#                       NIS policy using settings provided by the first
#                       iterface and service that provided it.
#  - "<interface name>" as yes, but only using settings from interface.
#
NETCONFIG_NIS_SETDOMAINNAME="yes"
```

When wicked is up, netconfig sets domainname form what DHCP server offers or from the content of file `/etc/defaultdomain`. If I set `NETCONFIG_NIS_SETDOMAINNAME` to no, the domainname is set on boot.

**Comment by johnliu55tw** [comment]
@gitlawr Just tested on latest master ISO (2021-11-18), seems like `os.sysctls` works except for the special case of `kernel.domainname` as @bk201 mentioned above. Do you think we still need to fix this?

**Comment by gitlawr** [comment]
We can close after we fix the doc(both v0.3 & dev):

![image](https://user-images.githubusercontent.com/5697937/142388958-9bcc18a6-303e-434b-9ccc-039a132664d6.png)

**Comment by lanfon72** [comment]
verified config option works correctly.

Test Information
----
* Environment: **qemu/KVM** single node
* Harvester Version: **master-b7e562d3-head**
